lua/wire/client/wire_listeditor.lua

```diff
@@ -112,42 +94,44 @@ local function fileName(filepath)
 end
 
 local function SaveTo(self, func, ...)
-	if (!IsValid(self)) then return end
+	if not IsValid(self) then return end
 	local args = {...}
 
 	local path = self.FileBrowser:GetFileName() or self.m_strList or ""
 
 	Derma_StringRequestNoBlur(
 		"Save to New File",
 		"",
-		string.sub(fileName(path), 0, -5), // remove .txt at the end
+		string.sub(fileName(path), 0, -5), -- remove .txt at the end
```

Comment:
Suggestion; Use pattern to find the length of file extension, and then substring with that length - 1 (to return a filename without extension).
(Just in case, is it safe to guarantee an extension will always be ".txt")